nixos/tests/gns3-server.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, ... }:
+
+pkgs.testers.runNixOSTest ({ pkgs, lib, ... }: {
```

Comment:
Yes I used that notation because I am testing this GNS3 NixOS module in my nur-packages repository, and this is the only way I found to make the test work on nixpkgs and in my nur-packages repository.

But indeed looks like the convention is to use `import ./make-test-python.nix`, so I reverted my change :)